sql/src/main/java/io/crate/execution/dsl/projection/ColumnIndexWriterProjection.java

```diff
@@ -127,16 +125,6 @@ public boolean isIgnoreDuplicateKeys() {
         return visitor.visitColumnIndexWriterProjection(this, context);
     }
 
-    @Override
-    public void replaceSymbols(Function<? super Symbol, ? extends Symbol> replaceFunction) {
-        Lists2.replaceItems(columnSymbols, replaceFunction);
```

Comment:
not so related for lists, but I maybe we should make all these instance vars final now as well